runelite-client/src/main/java/net/runelite/client/plugins/StatusBars/StatusBarsOverlay.java

```diff
@@ -0,0 +1,219 @@
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.StatusBars;
+
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+import java.awt.Graphics2D;
+import java.awt.Dimension;
+import java.awt.Image;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Client;
+import net.runelite.api.Point;
+import net.runelite.api.Skill;
+import net.runelite.api.VarPlayer;
+import net.runelite.api.widgets.Widget;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import javax.inject.Inject;
+import net.runelite.client.game.SkillIconManager;
+
+@Slf4j
+public class StatusBarsOverlay extends Overlay
+{
+	// Static colors
+	final static Color DEPLETED = new Color(25, 25, 25, 150);
+	final static Color PRAYER_COLOR = new Color(50, 200, 200, 150);
+	final static Color BACKGROUND = new Color(0, 0 , 0, 150);
+	final static Color HEALTH_COLOR = new Color(200, 35, 0, 150);
+	final static Color POISONED_COLOR = new Color(0, 145, 0, 150);
+	final static Color VENOMED_COLOR = new Color(0, 65, 0, 150);
+	final static Color COUNTER_COLOR = new Color(255, 255,  255, 255);
+	final static Color NO_COLOR = new Color(0, 0, 0, 0);
+	final private static int HEIGHT = 252;
+	final private static int WIDTH = 20;
+	final private static int PADDING = 1;
+	private Client client;
+	private StatusBarsConfig config;
+	private SkillIconManager skillIconManager;
+	private FontManager fontManager;
+	private Color healthBar;
+	private Color counter;
+
+	@Inject
+	public StatusBarsOverlay(Client client, StatusBarsConfig config, SkillIconManager skillIconManager)
+	{
+		setLayer(OverlayLayer.ABOVE_WIDGETS);
+		this.client = client;
+		this.config = config;
+		this.skillIconManager = skillIconManager;
+	}
+
+	@Override
+	public Dimension render(Graphics2D g)
+	{
+		renderInventoryBars(g);
+		return null;
+	}
+
+	private void renderInventoryBars(Graphics2D g)
```

Comment:
This is just called from render, can it be inlined there?

Reply:
i'm not sure what you want me to do hmm

Reply:
allright, i did it and it still works. great :)